src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -54,31 +66,43 @@ private void loadData() {
 	
 		
 		// Nintendo
-		listNintendo = new ArrayList<Nintendo>();
+		//Create Faculties
```

Comment:
I think the code is missing a lot of comments, which may affect the later work due to confusion, so I added some comments.